<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateClient</class>
 <widget class="QMainWindow" name="CreateClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblClientIDprmpt">
      <property name="text">
       <string>ClientID:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblClientUsernameprmpt">
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lblClientCPUCountprmpt">
      <property name="text">
       <string>CPUCount</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lblClientCPUInfo">
      <property name="text">
       <string>CPUInfo</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lblClientPlatformprmpt">
      <property name="text">
       <string>Platform</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblClientWorkCompletedprmpt">
      <property name="text">
       <string>WorkCompleted</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblClientPassword">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtClientUsername"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="lblClientID">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtClientPassword"/>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="lblClientWorkCompleted">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="numClientCPUCount">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>126</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="txtClientCPUInfo"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="txtClientPlatform"/>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="btnCreateClient">
      <property name="text">
       <string>Create Client Profile</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>372</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
